src/components/edge-auth0-sync.vue

```diff
@@ -0,0 +1,231 @@
+          >
+            <v-icon
+              style="margin: 0.5rem 0;"
+              center
+              id="close-icon"
+              size="25"
+            >
+              mdi-close
+            </v-icon>
+          </div>
+        </div>
+
+        <hr>
+
+        <div>
+          <div v-if="EDGE_SYNC_COMPATIBILITY_STATUS === 'CHECKING'">
+            <br>
+            <v-progress-circular
+              indeterminate
+              :width="3.5"
+              id="compatibile-spinner"
+              :size="40"
+              color="primary"
+            />
+            <br>
+            <br>
+            <p id="checker-text">
+              Checking if your connected Edge Device is compatible with Ambianic Premium Services.
+            </p>
+            <br>
+          </div>
+
+          <v-list v-if="EDGE_SYNC_COMPATIBILITY_STATUS === 'OUTDATED'">
+            <br>
+            <div class="align-center">
+              <v-icon
+                style="margin: 0;"
+                center
+                id="outdated-icon"
+                size="45"
+              >
+                mdi-alert-outline
+              </v-icon>
+            </div>
+            <br>
+            <p id="outdated-text">
+              Your Edge Device is currently running on version {{ edgeVersion }} and is missing the new <a href="https://docs.ambianic.ai/users/premium-services/">Premium Services</a> feature.
+            </p>
+            <br>
+
+            <hr>
+            <div
+              class="flex"
+              style="margin: .5rem 0 0 0"
+            >
+              <div class="align-center">
+                <p style="margin: 0 .3rem; font-size: .95rem">
+                  Please restart your Ambianic Edge device to update your Ambianic Edge Installation.
+                </p>
+              </div>
+            </div>
+          </v-list>
+
+          <div v-if="EDGE_SYNC_COMPATIBILITY_STATUS === 'COMPATIBLE'">
+            <v-list v-if="!isEdgeSynced">
+              <br>
+              <v-progress-circular
+                indeterminate
+                :width="3.5"
+                id="loading-sync"
+                :size="40"
+                color="primary"
+              />
+              <br>
+              <br>
+              <p id="loading-explanation">
+                Your premium subscription is being extended to your connected Edge Device.
+              </p>
+              <br>
+            </v-list>
+
+            <v-list v-else>
+              <v-icon
+                style="margin: 0.5rem 0;"
+                center
+                id="success-icon"
+                size="60"
+              >
+                mdi-check-circle-outline
+              </v-icon>
+
+              <p id="explanation">
+                Your premium subscription has been successfully extended to your connected Edge Device. <br> <br> You would now get email notifications about detections events from your running edge device.
```

Comment:
@vickywane a small language correction. Please replace:
"You would now get email notifications about detections events from your running edge device." with

"You will  now receive email notifications about detections events from your running edge device."

Reply:
I have made this correction